src/Microsoft.OData.Edm/Schema/EdmContainedEntitySet.cs

```diff
@@ -179,7 +179,7 @@ private IEdmPathExpression ComputePath()
                 newPath.Add(targetType.FullTypeName());
             }
 
-            newPath.Concat(this.NavigationPath.PathSegments);
+            newPath.AddRange(this.NavigationPath.PathSegments);

```

Comment:
What is the difference between the AddRange and Concat functions on a generic List

Reply:
Concat returns a new sequence, but we never assigned the result to a variable so it was effectively a no-op. So we could either do:
   newPath = newPath.Concat(...)
or simply:
   newPath.AddRange(...)